### ISSUE TITLE
Inquiries Regarding VoxBlink2 Dataset and Training Methods

### ISSUE THREAD

**Issue by axuan731** [VoxBlink2/ScriptsForVoxBlink2#3]
Dear VoxBlink2 Team Members,

Thank you very much for providing the paper and dataset, which have significantly contributed to academic advancement. After reading your paper and code, I have a few questions I would like to ask.

1. It appears that many audio-video clips are repeated, for example:
```
Identity: id000002
Reference: YJUT3czS6OI
Start and Duration: 0.000 2.410
KBPS: 166.072
...
Identity: id000002
Reference: YJUT3czS6OI
Start and Duration: 0.000 4.120
KBPS: 157.988
...
```
It seems that both of these audio segments contain the duration from 0 to 2.410 seconds. There are many such instances. Is this correct? Could this negatively impact the training process?

2. You used a relatively high learning rate (0.1) during training. I previously trained audio data for 100,000 speakers with a similar learning rate, but the high learning rate caused my model to fail to converge. Consequently, I had to lower the learning rate and decrease the number of warm-up epochs. Would you mind sharing the specific parameters used during your training or any tips for training with large datasets?

3. You utilized speed perturbation during training as an augmentation strategy, which is typically used for smaller speaker datasets (such as VoxCeleb2). Have you tried training without speed perturbation during the initial training phase? Do you believe this augmentation strategy remains effective with such a large dataset?

Thank you for your time and assistance!^_^

**Comment by VoxBlink2** [comment]
Fine!
1. Just ignore the 【Start and Duration】since the 【start】 timestamps sometimes are mislabeled when the start time equals 0.0 (We are going to fix the bug). However, the recipes we provided use the start and end of the video-frames to crop audio segments, so just follow the recipe. ^^ That is correct.
2. In order to make more precise adjustments, we use the StepLR scheduler. For small models like ResNet34, the step size is set 4-6 and then reduces to one tenth. While it comes to larger models, sometimes the step size is set 1-3(depends on the model size) and we finally early-stopped the pre-training phases. Eg. we use the 11th epoch of SimAM-ResNet293 for finetuning. Additionally, we used large-scale distributed training supported by the supercomputing techniques. Therefore, the larger batch size(1024 or more) may help accelerate the training process. Basically, the scale of your training data needs to match the size of the model. 
3. For smaller model, there is no significant difference between w and w/o speed perturbation. But we found speed perturbation seems benefit the performance of larger model pre-training. In the fine-tuning stage, adopting speed perturbation and a smaller LR is effective in our experiment.
I hope these prove useful to you.^^

**Comment by axuan731** [comment]
> Fine!
> 
> 1. Just ignore the 【Start and Duration】since the 【start】 timestamps sometimes are mislabeled when the start time equals 0.0 (We are going to fix the bug). However, the recipes we provided use the start and end of the video-frames to crop audio segments, so just follow the recipe. ^^ That is correct.
> 2. In order to make more precise adjustments, we use the StepLR scheduler. For small models like ResNet34, the step size is set 4-6 and then reduces to one tenth. While it comes to larger models, sometimes the step size is set 1-3(depends on the model size) and we finally early-stopped the pre-training phases. Eg. we use the 11th epoch of SimAM-ResNet293 for finetuning. Additionally, we used large-scale distributed training supported by the supercomputing techniques. Therefore, the larger batch size(1024 or more) may help accelerate the training process. Basically, the scale of your training data needs to match the size of the model.
> 3. For smaller model, there is no significant difference between w and w/o speed perturbation. But we found speed perturbation seems benefit the performance of larger model pre-training. In the fine-tuning stage, adopting speed perturbation and a smaller LR is effective in our experiment.
>    I hope these prove useful to you.^^

Thank you very much for your response. It will be of great help to my future research!
